easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.3.0-intel-2018a.eb

```diff
@@ -0,0 +1,52 @@
+# easyconfig file for Code_Saturne
+# author: Metin Cakircali (Juelich Supercomputing Centre)
+name = 'Code_Saturne'
+version = '5.3.0'
```

Comment:
Is there a reason to have `5.3.0` and `5.0.9` installed at the same time? If not, I'd suggest to remove `5.0.9` from the PR

Reply:
5.0.9 is long term support, 5.3.0 is latest..